### PR TITLE
Separated Lint tests

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,9 @@ python:
   - "2.7"
 
 env:
-  - VERSION="7.0" ODOO_REPO="odoo/odoo"
-  - VERSION="7.0" ODOO_REPO="OCA/OCB"
+  - VERSION="8.0" LINT_CHECK="1"
```
Comment (pedrobaeza) [inline .travis.yml:7]: Although it's not strictly needed, put 7.0 for coherence.

Comment (yvaucher) [inline .travis.yml:7]: Hum we are on 8.0 branch there. Shouldn't I change travis to point on 8.0 instead ?

Comment (pedrobaeza) [inline .travis.yml:7]: Oh, good point, I didn't see destination branch. Yeah, all must go to 8.0.

**Comment by guewen** [comment]
:+1:

**Pull request merged**